### PR TITLE
deps: unpin msgpack dependency

### PR DESCRIPTION
**Pull request by owenpearson**
see https://github.com/ably/ably-ruby/issues/388

### CODE REVIEW

File: ably-rest.gemspec [hunk 1]
```diff
@@ -37,7 +37,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'faraday-typhoeus', '~> 0.2.0'
 
   spec.add_runtime_dependency 'json'
-  spec.add_runtime_dependency 'msgpack', '=1.5.2'
+  spec.add_runtime_dependency 'msgpack', '>= 1.5.2'
```
Comment (lawrence-forooghian) [inline ably-rest.gemspec:40]: I don't think we want to allow major versions greater than 1, given that they might introduce a breaking API change and hence no longer function with our SDK. How about:

```suggestion
  spec.add_runtime_dependency 'msgpack', '>= 1.5.2', '< 2'
```
```

Comment (owenpearson) [reply ably-rest.gemspec]: nice spot, updated

Comment (lawrence-forooghian) [reply ably-rest.gemspec]: 👍, looks like some indentation has gone missing now though?

Comment (owenpearson) [reply ably-rest.gemspec]: oops, fixed

**Review by lawrence-forooghian — APPROVED** [review]

**Pull request merged**